doc/importexport.rst

```diff
@@ -239,17 +239,86 @@ optimum values that will produce an acceptable mesh.
 Exporting DXF
 ##############
 
+.. seealso::
+
+    :class:`cadquery.occ_impl.exporters.dxf.DxfDocument` for exporting multiple
+    Workplanes to one or many layers of a DXF document.
+
+Options
+-------
+
+``approx``
+    Approximation strategy for converting :class:`cadquery.Workplane` objects to DXF entities:
+
+        ``None``
+            no approximation applied
+        ``"spline"``
+            all splines approximated as cubic splines
+        ``"arc"``
+            all curves approximated as arcs and straight segments
+``tolerance``
+    Approximation tolerance for converting :class:`cadquery.Workplane` objects to DXF entities.
+    See `Approximation strategy`_.
+``doc_units``
+    Ezdxf document/modelspace :doc:`units <ezdxf-stable:concepts/units>`.
+    See `Approximation strategy`_.
+
+.. code-block:: python
+    :caption: DXF document without options.
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.exportDXF(result, "/path/to/file/object.dxf")
+    # or
+    exporters.export(result, "/path/to/file/object.dxf")
+
+
+Units
+-----
+
+The default DXF document units are mm (:code:`doc_units = 4`), however the document units can be set to any
+:doc:`unit supported by ezdxf <ezdxf-stable:concepts/units>`.
+
```

Comment:
@sethfischer How about including a quick reference of common units directly say 0 to 6?
```
The default DXF document units are mm (:code:`doc_units = 4`).

========= ===============
doc_units Unit
========= ===============
0         Unitless
1         Inches
2         Feet
3         Miles
4         Millimeters
5         Centimeters
6         Meters
========= ===============

Document units can be set to any :doc:`unit supported by ezdxf <ezdxf-stable:concepts/units>`.
```